src/api-docs.js

```diff
@@ -786,6 +786,100 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/tx-proposal/melt-tokens': {
+      post: {
+        summary: 'Get the hex representation of a melt tokens transaction without input data.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            in: 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          },
+        ],
+        requestBody: {
+          description: 'Data to melt tokens.',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['token', 'amount'],
+                properties: {
+                  token: {
+                    type: 'string',
+                    description: 'UID of the token to melt.'
+                  },
+                  amount: {
+                    type: 'integer',
+                    description: 'The amount of tokens to melt. It must be an integer with the value in cents, i.e., 123 means 1.23.'
+                  },
+                  deposit_address: {
+                    type: 'string',
+                    description: 'Optional deposit_address to send the deposit HTR received after the melt.'
+                  },
+                  change_address: {
+                    type: 'string',
+                    description: 'Optional address to send the change amount of custom tokens after melt.'
+                  },
+                  melt_authority_address: {
+                    type: 'string',
+                    description: 'Optional address to send the new melt authority output created.'
+                  },
+                  allow_external_melt_authority_address: {
+                    type: 'boolean',
+                    description: 'If the melt authority address is allowed to be from another wallet. Default is false.'
+                  },
+                }
+              },
+              examples: {
+                data: {
+                  summary: 'Data to melt tokens.',
+                  value: {
+                    token: '000016392ed330ed99ff0f74e4169a8d257fd1d07d3b38c4f8ecf21a78f10efa',
+                    amount: 100,
+                  }
+                }
+              }
+            }
+          }
+        },
+        responses: {
+          200: {
+            description: 'Melt tokens.',
+            content: {
+              'application/json': {
+                examples: {
+                  error: {
+                    summary: 'Insuficient amount of tokens',
+                    value: { success: false, error: "There aren't enough inputs to melt." }
```

Comment:
I suggest mentioning the tokens within the inputs to avoid confusion for the reader.
```suggestion
                    value: { success: false, error: "There aren't enough tokens in the inputs to melt." }
```